bench-tool/src/BenchTool.hs

```diff
@@ -18,7 +19,7 @@ import Network.GRPC.HTTP2.Encoding as Encoding
 
 -- client
 import Network.GRPC.Client (open, singleRequest, streamReply, streamRequest, Timeout(..), StreamDone(..), CompressMode(..))
-import qualified Network.HTTP2.Client as Client
+import qualified "http2-client" Network.HTTP2.Client as Client
```

Comment:
What is the reason for using the package imports extension?

Why is it needed now, but was not needed before?

We generally don't use this extension, so an argument must be made as to why it is necessary.

Reply:
There was a name conflict

Reply:
The cause is that in version 3 of this package https://hackage.haskell.org/package/http2-3.0.3/docs/Network-HTTP2-Client.html the `Client` module was added. And LTS-19.6 has version 3, and it had version <3 in the previous LTS we used.